README.md

```diff
@@ -31,72 +31,582 @@ _Kev_ reduces the need for Kubernetes expertise in the team. The generated Kuber
 
 * **Easy integrations** - You may use generated Kubernetes manifests with any tool / framework of your choice. We aim at adding some useful integrations further improving developer experience.
 
-## Quick Start
+## Installation
 
-All you need to get started quickly is [kev](https://github.com/appvia/kev/releases) binary added to your PATH, and one or more docker compose files.
+All you need to get started quickly is the [kev](https://github.com/appvia/kev/releases) binary added to your PATH, and one or more docker compose files.
 
-### Initiate a project
+## Quickstart
 
-Run the following command from within your project directory to initialise Kev project:
+- `kev init` - identifies a project's Compose Kubernetes source files and creates Compose environment overrides.
+- `kev render` - detects, applies any config changes and generates deployment manifests.
+- `kev help` - run it if you're a little lost.
+
+### init
+
+Run the following command within your project directory:
 
 ```sh
 kev init
 ```
 
-The command above will auto-detect default `docker-compose.yaml` and `docker-compose.override.yaml` files (if present) in the project directory, and tracks them as Kubernetes deployment sources.
+This identifies the default `docker-compose.yaml` and (if present) `docker-compose.override.yaml` files in your project directory. They will be used as Compose Kubernetes sources.
```

Comment:
need to clarify that so it's less confusing... `as Compose Kubernetes sources` -> perhaps `as source of truth for your application deployment in Kubernetes`?